internal/test/scorecard/tests.go

```diff
@@ -101,28 +99,6 @@ func CryostatConfigChangeTest(bundle *apimanifests.Bundle, namespace string, ope
 	defer cleanupCryostat(r, tr.Client, CryostatConfigChangeTestName, namespace)
 
 	// Switch Cryostat CR to PVC for redeployment
-	ctx, cancel := context.WithTimeout(context.Background(), testTimeout)
-	defer cancel()
-	client := tr.Client
-
-	cr, err = client.OperatorCRDs().Cryostats(namespace).Get(ctx, CryostatConfigChangeTestName)
-	if err != nil {
-		return fail(*r, fmt.Sprintf("failed to get Cryostat CR: %s", err.Error()))
-	}
-	cr.Spec.StorageOptions = &operatorv1beta1.StorageConfiguration{
-		PVC: &operatorv1beta1.PersistentVolumeClaimConfig{
-			Spec: &corev1.PersistentVolumeClaimSpec{
-				StorageClassName: nil,
-				Resources: corev1.ResourceRequirements{
-					Requests: corev1.ResourceList{
-						corev1.ResourceStorage: resource.MustParse("1Gi"),
-					},
-				},
-			},
-		},
-	}
-
-	// Wait for redeployment of Cryostat CR
 	err = updateAndWaitTillCryostatAvailable(cr, tr)
```

Comment:
I am seeing this failure. The `cr` here is not updated and is actually just an empty struct, thus the `status.ApplicationURL` is `""`

```
Image:      quay.io/thvo/cryostat-operator-scorecard:2.5.0-20240326201336
Entrypoint: [cryostat-scorecard-tests cryostat-config-change]
Labels:
	"suite":"cryostat"
	"test":"cryostat-config-change"
Results:
	Name: cryostat-config-change
	State: fail

	Errors:
		failed to reach the application: Get "health": unsupported protocol scheme ""
	Log:
		deployment cryostat-config-change is not yet found
		deployment cryostat-config-change is not yet found
		deployment cryostat-config-change is not yet found
		deployment cryostat-config-change is not yet found
		deployment cryostat-config-change is not yet found
		deployment cryostat-config-change is not yet found
		deployment cryostat-config-change is not yet available
		deployment cryostat-config-change is not yet available
		deployment cryostat-config-change is not yet available
		deployment cryostat-config-change is not yet available
		deployment cryostat-config-change is not yet available
		deployment cryostat-config-change is not yet available
		deployment cryostat-config-change is not yet available
		deployment cryostat-config-change is not yet available
		deployment cryostat-config-change is not yet available
		deployment cryostat-config-change is not yet available
		deployment cryostat-config-change is not yet available
		deployment cryostat-config-change is available
		application is available at https://testing.cryostat
		Cryostat deployment is still updating. Storage: &EmptyDirVolumeSource{Medium:,SizeLimit:0,}
		Waiting for deployment cryostat-config-change rollout to finish: 0 out of 1 new replicas have been updated... 
		Waiting for deployment cryostat-config-change rollout to finish: 0 out of 1 new replicas have been updated... 
		Waiting for deployment cryostat-config-change rollout to finish: 0 out of 1 updated replicas are available... 
		Waiting for deployment cryostat-config-change rollout to finish: 0 out of 1 updated replicas are available... 
		Waiting for deployment cryostat-config-change rollout to finish: 0 out of 1 updated replicas are available... 
		Waiting for deployment cryostat-config-change rollout to finish: 0 out of 1 updated replicas are available... 
		Waiting for deployment cryostat-config-change rollout to finish: 0 out of 1 updated replicas are available... 
		Waiting for deployment cryostat-config-change rollout to finish: 0 out of 1 updated replicas are available... 
		Waiting for deployment cryostat-config-change rollout to finish: 0 out of 1 updated replicas are available... 
		Waiting for deployment cryostat-config-change rollout to finish: 0 out of 1 updated replicas are available... 
		Waiting for deployment cryostat-config-change rollout to finish: 0 out of 1 updated replicas are available... 
		Waiting for deployment cryostat-config-change rollout to finish: 0 out of 1 updated replicas are available... 
		Waiting for deployment cryostat-config-change rollout to finish: 0 out of 1 updated replicas are available... 
		Waiting for deployment cryostat-config-change rollout to finish: 0 out of 1 updated replicas are available... 
		Waiting for deployment cryostat-config-change rollout to finish: 0 out of 1 updated replicas are available... 
		Waiting for deployment cryostat-config-change rollout to finish: 0 out of 1 updated replicas are available... 
		deployment cryostat-config-change successfully rolled out
		Cryostat deployhttps://docs.github.com/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntaxment has successfully updated with new spec template
```